src/Dodo.HttpClient.ResiliencePolicies/HttpClientSettings.cs

```diff
@@ -1,46 +1,46 @@
-﻿using System;
+using System;
 using Dodo.HttpClientResiliencePolicies.CircuitBreakerSettings;
 using Dodo.HttpClientResiliencePolicies.RetrySettings;
 
 namespace Dodo.HttpClientResiliencePolicies
 {
 	public class HttpClientSettings
 	{
-		public TimeSpan HttpClientTimeout { get; }
 		public TimeSpan TimeoutPerTry { get; }
+		public TimeSpan TimeoutOverall { get; }
 		public IRetrySettings RetrySettings { get; }
 		public ICircuitBreakerSettings CircuitBreakerSettings { get; }
 
-
 		public HttpClientSettings(
-			TimeSpan httpClientTimeout,
 			TimeSpan timeoutPerTry,
-			int retryCount) : this(httpClientTimeout, timeoutPerTry,
+			int retryCount,
+			TimeSpan timeoutOverall) : this(timeoutPerTry,
```

Comment:
Preserver order in the constructors: Move TimeoutOverall as first argument (because HttpClientTimeout was the first),

Reply:
Agree. Done.